apps/mobile/src/libs/components/components.tsx

```diff
@@ -1,13 +1,15 @@
 export { App } from "./app/app";
 export { BackgroundWrapper } from "./background-wrapper/background-wrapper";
 export { Button } from "./button/button";
+export { Checkbox } from "./checkbox/checkbox";
 export { GradientText } from "./gradient-text/gradient-text";
 export { Input } from "./input/input";
 export { Link } from "./link/link";
 export { LoaderWrapper } from "./loader-wrapper/loader-wrapper";
 export { ScreenWrapper } from "./screen-wrapper/screen-wrapper";
 export { Text } from "./text/text";
 export { ToastMessage } from "./toast-message/toast-message";
+export { default as CustomCheckbox } from "@react-native-community/checkbox";
```

Comment:
If it will be used only in our custom checkbox component, you can move the import there

Reply:
Ok!